addons/mail/static/src/models/media_preview/media_preview.js

```diff
@@ -148,6 +148,29 @@ function factory(dependencies) {
             return this.videoStream !== null;
         }
 
+        /**
+         * @private
+         * @returns {number}
+         */
+        _computeVideoElementHeight() {
+            return this.videoElementWidth * .75;
+        }
+
+        /**
+         * @private
+         * @returns {number}
+         */
+        _computeVideoElementWidth() {
+            const device = this.messaging && this.messaging.device;
+            if (!device) {
+                return 0;
+            }
+            if (device.isMobile) {
+                return device.globalWindowInnerWidth;
+            }
+            return device.globalWindowInnerWidth / 3;
```

Comment:
This might get a bit too small in "almost mobile" size (and probably way too big for people with massive widescreen), so this should have more thresholds for a better result, and a hard limit both up and down.